model/actors/miner/minerinfo.go

```diff
@@ -0,0 +1,54 @@
+package miner
+
+import (
+	"context"
+
+	"github.com/go-pg/pg/v10"
+	"go.opentelemetry.io/otel/api/global"
+	"golang.org/x/xerrors"
+)
+
+type MinerInfo struct {
```

Comment:
do we need a migration? this looks different from what i see in my current db schema

Reply:
Yeah we need quite a few (:crying_cat_face:), once I get those in, in the next hour or so, I'll take this out of draft.